private/aws-client-api-test/package.json

```
@@ -26,7 +25,7 @@
     "@types/node": "^12.7.5",
     "concurrently": "7.0.0",
     "downlevel-dts": "0.10.1",
-    "typedoc": "0.19.2",
+    "typedoc": "^0.23.23",
```

Suggestion: Either use pinned dependency of typedoc, or remove it entirely for private clients.
```suggestion
    "typedoc": "0.23.23",
```